### PR TITLE
Updated release notes for the adapter,version 3.17

### PR DESCRIPTION
**Pull request by OsirisTerje**

### CODE REVIEW

File: docs/articles/vs-test-adapter/Adapter-Release-Notes.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 ## NUnit3 Test Adapter for Visual Studio - Version 3.17.0 - July 11, 2020
 
-This release has a series of enhancements and bugfixes.  It has been preceeded by two beta releases (beta.1 and beta.2) published on nuget.org.
+This release has a series of enhancements and bugfixes.  There has been two 3.17 beta releases (beta.1 and beta.2) earlier published on nuget.org.
```
Comment (jnm2) [inline docs/articles/vs-test-adapter/Adapter-Release-Notes.md:5]:
```suggestion
This release has a series of enhancements and bugfixes.  There have been two 3.17 beta releases (beta.1 and beta.2) published earlier on nuget.org.
```

File: docs/articles/vs-test-adapter/Adapter-Release-Notes.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
 # Adapter Release Notes
 
+## NUnit3 Test Adapter for Visual Studio - Version 3.17.0 - July 11, 2020
+
+This release has a series of enhancements and bugfixes.  There has been two 3.17 beta releases (beta.1 and beta.2) earlier published on nuget.org.
```
Comment (jnm2) [inline docs/articles/vs-test-adapter/Adapter-Release-Notes.md]:
```suggestion
This release has a series of enhancements and bugfixes.  There have been two 3.17 beta releases (beta.1 and beta.2) published earlier on nuget.org.
```

File: docs/articles/vs-test-adapter/Adapter-Release-Notes.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+A major bugfix is that the console output now is properly coming out.  There are also a series of fixes for corner cases.
```
Comment (jnm2) [inline docs/articles/vs-test-adapter/Adapter-Release-Notes.md:6]: Does 'properly appearing' sound smoother?

File: docs/articles/vs-test-adapter/Adapter-Release-Notes.md [hunk 4]
```diff
@@ -1,5 +1,39 @@
 # Adapter Release Notes
 
+## NUnit3 Test Adapter for Visual Studio - Version 3.17.0 - July 11, 2020
+
+This release has a series of enhancements and bugfixes.  There have been two 3.17 beta releases (beta.1 and beta.2) published earlier on nuget.org.
+A major bugfix is that the console output now is properly coming out.  There are also a series of fixes for corner cases.  
+
+[StopOnError](https://docs.nunit.org/articles/vs-test-adapter/Tips-And-Tricks.html#stoponerror) can be very useful to reduce time of long test runs that are failing.
+
+The ability to set custom paths for the test code can be useful with data driven tests, where you want to point to the data file (and line number therein) instead of usually trivial test case.
```
Comment (jnm2) [inline docs/articles/vs-test-adapter/Adapter-Release-Notes.md:10]: What does "instead of usually trivial test case" part mean?

File: docs/articles/vs-test-adapter/Adapter-Release-Notes.md (hunk 3 above)
Comment (OsirisTerje) [reply docs/articles/vs-test-adapter/Adapter-Release-Notes.md:6]: Done

File: docs/articles/vs-test-adapter/Adapter-Release-Notes.md (hunk 4 above)
Comment (OsirisTerje) [reply docs/articles/vs-test-adapter/Adapter-Release-Notes.md]: Changed.....

File: docs/articles/vs-test-adapter/Adapter-Release-Notes.md [hunk 5]
```diff
@@ -1,5 +1,40 @@
 # Adapter Release Notes
 
+## NUnit3 Test Adapter for Visual Studio - Version 3.17.0 - July 11, 2020
+
+This release has a series of enhancements and bugfixes.  There have been two 3.17 beta releases (beta.1 and beta.2) published earlier on nuget.org.
+A major bugfix is that the console output now is properly appearing.  There are also a series of fixes for corner cases.  
+
+[StopOnError](https://docs.nunit.org/articles/vs-test-adapter/Tips-And-Tricks.html#stoponerror) can be very useful to reduce time of long test runs that are failing.
+
+The ability to set custom paths for the test code can be useful with data driven tests, where you want to point to the data file (and line number therein) instead of the default C# test code.
```
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Adapter-Release-Notes.md:10]:
```suggestion
The ability to set custom paths for the test code can be useful with data driven tests, where you want to point to the data file (and line number therein) instead of the default C# test code.
```

Suggestion: removing a trailing space (fixes markdown linting).

**Comment by OsirisTerje** [comment]
@SeanKilleen  Can you merge?  Release is today.

**Comment by SeanKilleen** [comment]
@OsirisTerje done!
